pow/src/main/java/tech/pegasys/artemis/ganache/GanacheController.java

```diff
@@ -44,7 +44,7 @@
   private String provider;
   private int waitIterations;
 
-  private static final ALogger LOG = new ALogger();
+  private volatile File keyFile;
```

Comment:
Why did we need to make the keyFile volatile here? is it because the file might get deleted as we're running Artemis?

Reply:
The variable is accessed from multiple threads so it has to be volatile to ensure they have a consistent view of its current value (or use synchronised blocks around where it's accessed).